config/jobs/kubernetes/sig-cloud-provider/gcp/gcp-gce.yaml

```diff
@@ -814,13 +814,12 @@ periodics:
       - --extract=ci/fast/latest-fast
       - --extract-ci-bucket=k8s-release-dev
       - --gcp-master-image=gci
-      - --gcp-node-image=gci
-      - --gcp-nodes=4
       - --gcp-zone=us-west1-b
       - --ginkgo-parallel=30
       - --provider=gce
       - --test_args=--ginkgo.skip=\[Driver:.gcepd\]|\[Slow\]|\[Serial\]|\[Disruptive\]|\[Flaky\]|\[Feature:.+\] --minStartupPods=8
-      - --timeout=50m
+      - --timeout=120m
```

Comment:
How did you arrive to 120m to be the threshold to run the testsuite with a single node?

Reply:
The release branch jobs that run the same set of test args are running fine with this timeout of 120m on single node cluster.
Please see https://github.com/kubernetes/test-infra/blob/master/releng/test_config.yaml#L332:L343
and
https://github.com/kubernetes/test-infra/blob/master/config/jobs/kubernetes/generated/generated.yaml#L61:L72